src/main/java/org/kitteh/irc/client/library/implementation/NettyManager.java

```diff
@@ -49,31 +83,12 @@
 import io.netty.handler.timeout.IdleStateHandler;
 import io.netty.util.CharsetUtil;
 import io.netty.util.concurrent.ScheduledFuture;
-import org.kitteh.irc.client.library.event.client.ClientConnectionClosedEvent;
-import org.kitteh.irc.client.library.exception.KittehConnectionException;
-import org.kitteh.irc.client.library.util.QueueProcessingThread;
-import org.kitteh.irc.client.library.util.ToStringer;
-
-import javax.annotation.Nonnull;
-import javax.annotation.Nullable;
-import javax.net.ssl.SSLException;
-import javax.net.ssl.TrustManagerFactory;
-import java.io.File;
-import java.io.IOException;
-import java.net.SocketAddress;
-import java.security.KeyStore;
-import java.security.KeyStoreException;
-import java.security.NoSuchAlgorithmException;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Queue;
-import java.util.Set;
-import java.util.concurrent.ConcurrentLinkedQueue;
-import java.util.concurrent.TimeUnit;
 
 final class NettyManager {
+
```

Comment:
Looks like your IDE is conflicting with Mbaxter's. I don't really care, but it might be useful to ensure you've got the same code style configuration as him later.